### PR TITLE
Documentation(847270): Add the column-rendering topics in column

### PR DESCRIPTION
**Pull request by Gayathri4135**
Add the define column manually and autogenerated columns topics in column rendering

### CODE REVIEW

File: ej2-javascript/code-snippet/grid/column-autogenerated-cs1/es5-datasource.js
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-autogenerated-cs1/es5-datasource.js:1]: remove unnecessary data like employeeData

File: ej2-javascript/code-snippet/grid/column-autogenerated-cs1/datasource.ts
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-autogenerated-cs1/datasource.ts:1]: remove unnecessary data

File: ej2-javascript/code-snippet/grid/column-autogenerated-cs1/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        { field: 'CustomerID', headerText: 'Customer ID', width: 120 },
+        { field: 'Freight', headerText: 'Freight', width: 100 ,format:'C2'},
+        { field: 'OrderDate', headerText: 'OrderDate', width: 100, format:'yMd'}
+    ],
+    height: 270
+});
+grid.appendTo('#Grid');
+grid.dataBound = () => {
+    const column = grid.columns[0];
+    (column as any).isPrimaryKey = 'true';
```
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-autogenerated-cs1/index.ts:17]: don't use any typing on column, please use respective model class

File: ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/datasource.ts
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/datasource.ts:1]: remove unnecessary data

File: ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/es5-datasource.js
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/es5-datasource.js:1]: remove unnecessary data

File: ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/index.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        { field: 'CustomerID', headerText: 'Customer ID', width: 120 },
+        { field: 'Freight', headerText: 'Freight', width: 100 ,format:'C2'},
+        { field: 'OrderDate', headerText: 'OrderDate', width: 100, format:'yMd'}
+    ],
+    pageSettings: { pageSize: 8 },
+  rowDataBound: (args: RowDataBoundEventArgs) => {
+    var rowIndex: number = parseInt(args.row.getAttribute('aria-rowIndex'));
+    var currentPageNumber: number = grid.pageSettings.currentPage;
+    var pageSize: number = grid.pageSettings.pageSize;
+    debugger;
```
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/index.ts:21]: remove debugger

File: ej2-javascript/code-snippet/grid/column-rendering-cs1/datasource.ts
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-rendering-cs1/datasource.ts:1]: remove unnecessary data

File: ej2-javascript/code-snippet/grid/column-rendering-cs1/es5-datasource.js
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/column-rendering-cs1/es5-datasource.js:1]: remove unnecessary data

File: ej2-javascript/code-snippet/grid/array-of-string-cs1/index.js
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/array-of-string-cs1/index.js:1]: Need to check below topics demo
Technical code changes are fine

File: ej2-javascript/code-snippet/grid/column-autogenerated-cs1/es5-datasource.js
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-autogenerated-cs1/es5-datasource.js:1]: removed

File: ej2-javascript/code-snippet/grid/column-autogenerated-cs1/datasource.ts
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-autogenerated-cs1/datasource.ts:1]: removed

File: ej2-javascript/code-snippet/grid/column-autogenerated-cs1/index.ts (hunk 1 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-autogenerated-cs1/index.ts]: Changed, using ColumnModel typing on column

File: ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/datasource.ts
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/datasource.ts:1]: removed

File: ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/es5-datasource.js
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/es5-datasource.js:1]: removed

File: ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/index.ts (hunk 2 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-dynamic-generation-cs1/index.ts]: removed

File: ej2-javascript/code-snippet/grid/column-rendering-cs1/datasource.ts
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-rendering-cs1/datasource.ts:1]: removed

File: ej2-javascript/code-snippet/grid/column-rendering-cs1/es5-datasource.js
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/column-rendering-cs1/es5-datasource.js:1]: removed

**Comment by Gayathri4135** [comment]
Changes moved to new hotfix branch https://github.com/syncfusion-content/ej2-grid-docs/pull/262  so i closed this PR

**Pull request closed without merging**